Dockerfile

```diff
@@ -96,6 +95,12 @@ RUN \
     && echo 'Cleaning up installation files' >&2 && \
     apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+# Install CMake 3.10
+RUN mkdir -p /opt && \
+    wget -q https://cmake.org/files/v3.10/cmake-3.10.0.tar.gz -O- \
+    | tar -C /opt -xz && cd /opt/cmake-3.10.0/ && ./bootstrap && \
+    make && make install && cd && rm -rf /opt/cmake-3.10.0
```

Comment:
why not use `/tmp` instead of `/opt`, as you delete it after installing anyway?

Reply:
To be honest, I just copied the section from below. But I can use /tmp instead. Let me try if this still works and then I will adjust.

Reply:
oh I oversaw that we use `/opt` already, mhm - either change both to `tmp` or stick with `opt`, but have it consistent, sorry for the noise.

Reply:
I think having it in `/tmp` is nice, because it may be mapped to RAM in the host system. /opt however is supposed to be persistent for the mips toolchain.

Reply:
@smlng the fixup commit changes that to /tmp, how do you feel about this?